src/elements/common/feature-checking/FeatureFlag.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { isFeatureEnabled, getFeatureConfig } from './util';
+import FeatureConsumer from './FeatureConsumer';
+import * as types from './flowTypes';
+
+interface FeatureFlagProps {
```

Comment:
```suggestion
export interface FeatureFlagProps {
```
then remove the export at the bottom of the file